platform/server-config/src/main/java/org/motechproject/server/config/SettingsFacade.java

```diff
@@ -215,6 +221,53 @@ public void saveRawConfig(String filename, String jsonText) {
     }
 
     /**
+     *  Allows persisting of raw JSON properties either in the database or file.
+     *
+     * @param filename json filename
+     * @param properties properties to persist
+     * @throws org.motechproject.commons.api.MotechException when I/O error occurs
+     */
+    public void saveRawConfig(String filename, Properties properties) {
```

Comment:
I don't see a reason for storing raw config as json, storing any format should be possible. In this case, I would store a .properties file as the raw config.